packages/block-editor/src/components/inserter/block-patterns.js

```diff
@@ -21,6 +21,35 @@ import InserterPanel from './panel';
 import { searchItems } from './search-items';
 import InserterNoResults from './no-results';
 
+const usePatternsState = ( onInsert ) => {
+	const { blockPatternCategories, patterns } = useSelect( ( select ) => {
+		const {
+			__experimentalBlockPatterns,
+			__experimentalBlockPatternCategories,
+		} = select( 'core/block-editor' ).getSettings();
+		return {
+			patterns: __experimentalBlockPatterns,
```

Comment:
For consistency it would be better to use `patterns` and `patternCategories` or both of them prefixed with `block`. I guess the shorter one is enough in this context.